### PR TITLE
Update production .war location

### PR DESCRIPTION
**Pull request by BarbaricCorgi**

### CODE REVIEW

File: pages/production.md [hunk 1]
```diff
@@ -37,8 +37,8 @@ Or when using Gradle:
 
 This will generate two files (if your application is called "jhipster"):
 
-*   `target/jhipster-0.0.1-SNAPSHOT.war`
```
Comment (pascalgrimaud) [inline pages/production.md]: we should keep both lines: target and build/libs, as it concerns Maven and Gradle too

Comment (BarbaricCorgi) [reply pages/production.md]: hmmm you're right indeed

**Comment by BarbaricCorgi** [comment]
I just fixed the paths to include both maven and gradle styles

**Comment by jdubois** [comment]
Great, thanks!

**Pull request merged**